portafly/src/i18n/locales/en/product.yml

```diff
@@ -0,0 +1,37 @@
+---
+# ------------------ Product > Create product
+# Page details
+
+page_title: Create product #--- PROPOSED
+page_title_description: Page title to create a product
+body_title: Create a new product #--- PROPOSED
+body_title_description: Body title of the page to create a new product
+
+# Main area
+
+name: Name
+name_description: Field label to show/input the name of the product.
+
+system_name:
```

Comment:
@porueesq @dfennessy As per https://github.com/3scale/porta/pull/2096 we have added inline help to the System name field to explain to users that a System name always needs to be unique.

Sorry for last minute change!

Reply:
Thanks for the heads up @cntlsn . I'll add the corresponding string.